static/templates/project/preview.html

```diff
@@ -0,0 +1,46 @@
+<div class="md-whiteframe-z1 _main-content _middle-content"
+     ng-class="{'_sm': $mdMedia('sm'), '_gt-sm': $mdMedia('gt-sm')}">
+    <div layout="row" class="layout-align-start-center _content-nav"
+         ng-class="{'_sm': $mdMedia('sm'), '_gt-sm': $mdMedia('gt-sm')}">
+        <md-button ui-sref="task_feed" class="_nav-actions" ng-disabled="taskfeed.previewedProject.status != 3">
+            <md-icon md-font-set="material-icons">chevron_left</md-icon>
+        </md-button>
+        <span class="_nav-title _truncated-text" style="max-width: 650px">{{ taskfeed.previewedProject.name }}</span>
+
+        <div class="_content-actions">
+
+
+        </div>
+    </div>
+    <div class=" _error-inset" style="margin: 16px 16px 16px 32px"
+         ng-if="project.project.status==project.status.STATUS_IN_PROGRESS || project.project.status==project.status.STATUS_PAUSED">
+        You are previewing this project. Click "Accept" to start working on it.
```

Comment:
That's fine for now. I need to figure out the broader situation of previewing...